services/worker/tests/job_runners/dataset/test_config_names.py

```diff
@@ -66,6 +67,21 @@ def test_compute(app_config: AppConfig, hub_public_csv: str, get_job_runner: Get
     assert len(content["config_names"]) == 1
 
 
+def test_compute_too_many_configs(app_config: AppConfig, get_job_runner: GetJobRunner) -> None:
+    dataset = "dataset"
+    max_number_of_configs = 2
+    job_runner = get_job_runner(
+        dataset,
+        replace(app_config, config_names=replace(app_config.config_names, max_number=max_number_of_configs)),
+    )
+
+    too_many_configs = ["config_1", "config_2", "config_3"]
+    with patch("worker.job_runners.dataset.config_names.get_dataset_config_names", return_value=too_many_configs):
```

Comment:
we should also check that it works for 1 and 2

Reply:
Done, actually, I changed the test to validate with less/equals and more configs than the max_number param.